single_kernel_mongo/managers/backups.py

```diff
@@ -324,7 +331,10 @@ def resync_config_options(self):  # pragma: nocover
             reraise=True,
         ):
             with attempt:
-                pbm_status = self.get_status()
+                pbm_statuses = self.get_statuses()
+                pbm_status = next(iter(pbm_statuses), None)
+
+                # todo future work - check status of pbm directly
```

Comment:
don't we already have `_get_backup_restore_operation_result` to get the information from?